tools/crs_converter/CRSconverter.xml

```diff
@@ -0,0 +1,278 @@
+            <filter>output_file_format and 'bmp' in output_file_format</filter>
+        </data>
+        <data name='shapefile' format='shp' from_work_dir='shapefile' label='Shapefile dataset'>
+            <filter>output_file_format and 'shp' in output_file_format</filter>
+        </data>
+    </outputs>
+    <tests>
+        <test expect_num_outputs='6'>
+            <param name='input_file' value='shapefiles/grid_aq_3_PF.html' ftype='shp'>
+                <composite_data value='shapefiles/grid_aq_3_PF.shp'/>
+                <composite_data value='shapefiles/grid_aq_3_PF.shx'/>
+                <composite_data value='shapefiles/grid_aq_3_PF.dbf'/>
+                <composite_data value='shapefiles/grid_aq_3_PF.prj'/>
+            </param>
+            <param name ='output_file_format' value='shp,pdf,png,jpeg,tiff,bmp'/>
+            <param name ='proj' value='laea'/>
+            <param name ='ellps' value='WGS84'/>
+            <param name ='lat_0' value='-50'/>
+            <param name ='lon_0' value='70'/>
+            <param name ='x_0' value='0'/>
+            <param name ='y_0' value='0'/>
+            <param name ='units' value='m'/>
+            <output file='results/PDF_output.pdf' name='PDF_output'>
+                <assert_contents>
+            	    <has_size value="26257" delta="0"/>
+            	</assert_contents>
+            </output>
+            <output file='results/PNG_output.png' name='png_output'>
+                <assert_contents>
+            	    <has_size value="54459" delta="0"/>
+            	</assert_contents>
+            </output>
+            <output file='results/JPEG_output.jpg' name='jpeg_output'>
+                <assert_contents>
+            	    <has_size value="47450" delta="0"/>
+            	</assert_contents>
+            </output>
+            <output file='results/TIFF_output.tiff' name='tiff_output'>
+                <assert_contents>
+            	    <has_size value="691360" delta="0"/>
+            	</assert_contents>
+            </output>
+            <output file='results/BMP_output.bmp' name='bmp_output'>
+                <assert_contents>
+            	    <has_size value="231478" delta="0"/>
+            	</assert_contents>
+            </output>
+            <output file='results/Shapefile_dataset_shp.html' name='shapefile' compare='sim_size' delta="0">
+                <extra_files type='file' name='shapefile.shp' value='results/shapefile.shp' compare='sim_size' delta="0"/>
+                <extra_files type='file' name='shapefile.shx' value='results/shapefile.shx' compare='sim_size' delta="0"/>
+                <extra_files type='file' name='shapefile.dbf' value='results/shapefile.dbf' compare='sim_size' delta="0"/>
+                <extra_files type='file' name='shapefile.prj' value='results/shapefile.prj' compare='sim_size' delta="0"/>
+            </output>
+        </test>
+    </tests>
+    <help><![CDATA[
+.. class:: infomark
+
+**What it does ?**
```

Comment:
I would add the links that you put in the little helps here too with a bit more info (so that the user don't have to switch pages every 2 min) 
I am not sure if this suggestion is the best solution :)